src/vmm_mad/remotes/kvm/deploy

```diff
@@ -34,8 +34,8 @@ DATA=`virsh --connect $LIBVIRT_URI create $DEP_FILE`
 
 if [ "x$?" = "x0" ]; then
 
-    DOMAIN_ID=$(echo $DATA | sed 's/Domain //' | sed 's/ created from .*$//' | tr -d "'")
-    UUID=$(virsh --connect $LIBVIRT_URI dominfo $DOMAIN_ID | grep UUID: | awk '{print $2}')
+    DOMAIN_ID=$(xmllint --xpath '/domain/name/text()' "$DEP_FILE")
```

Comment:
This would work but strictly speaking xmllint (xmllint2/libxml2-utils) is not in the packages dependency, so far we only have libxml2-dev which itself requires xmllint. Anyway, what about using single sed command like:
```
sed "s/Domain '\{0,1\}\([^ ']*\)'\{0,1\} created from deployment.*/\1/"
```
Should be faster than parsing the deployment file or is it too cryptic?

Reply:
My point is that _anything_ parsing a human-readable text is prone to fail because the text could change over the time, for example, due to a developer decision, enabled translation, etc...
IMO the domain XML is much more _stable_.

Reply:
ok, good point, I agree with the patch